test/config/SchemaValidationTest.js

```diff
@@ -40,6 +40,7 @@ describe('SchemaValidationTest', function () {
 
     it('invalid configuration is caught by the validator', function () {
         const invalidJson = fs.readFileSync(path.resolve(__dirname, 'configurations/invalid.json'), 'utf8');
-        expect(validateCandidate(invalidJson).errors[0]).to.exist.with.property('message', 'must have a minimum value of 1000');
+        expect(validateCandidate(invalidJson).errors[0]).to.exist.with.property('message');
```

Comment:
Not a great deal but we may do the `validateCandidate` call once, store it in some variable and do the assertions

Reply:
Yes, it makes sense. Let me fix it.

Reply:
Done.